File: sds_trabalho_1/server/model/person.py
class Person:
    def __init__(self, name, cpf, address):
        self.__name = name
        self.__cpf = cpf
        self.__address = address

    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name        

    def get_cpf(self):
        return self.__cpf

    def set_cpf(self, cpf):
        self.__cpf = cpf

    def get_address(self):
        return self.__address

    def set_address(self, address):
        self.__address = address

    def __str__(self):
        return f'Person{{cpf={self.__cpf}, name={self.__name}, address={self.__address}}}'

File: work-socket-client/src/main/java/dsd/socket/client/model/Person.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package dsd.socket.client.model;

/**
 *
 * @author Matheus
 */
public class Person {
    
    protected String cpf;
    protected String name;
    protected String address;
    protected Company company;
    
    public Person() {
    }

    public Person(String cpf, String name, String address) {
        this.cpf = cpf;
        this.name = name;
        this.address = address;
    }
    
    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public Company getCompany() {
        return company;
    }

    public void setCompany(Company company) {
        this.company = company;
        this.company.addPerson(this);
    }

    @Override
    public String toString() {
        StringBuilder companyString = new StringBuilder();

        if(company == null) {
            companyString.append("[]");
        } else {
            companyString = new StringBuilder()
                    .append("[")
                    .append(company.getId()).append(";")
                    .append(company.getCnpj()).append(";")
                    .append(company.getSocialReason()).append(";")
                    .append(company.getFoundedIn())
                    .append("]");
        }
        return new StringBuilder()
            .append(this.getClass().getSimpleName() + ";")
            .append(cpf + ";")
            .append(name + ";")
            .append(address + ";")
            .append(companyString)
            .toString();
    }
}
